third_party/proton/csrc/lib/Profiler/CuptiProfiler.cpp

```diff
@@ -173,10 +146,10 @@ struct CuptiProfiler::CuptiProfilerPimpl {
   static void callbackFn(void *userData, CUpti_CallbackDomain domain,
                          CUpti_CallbackId cbId, const void *cbData);
 
-  const inline static size_t AlignSize = 8;
-  const inline static size_t BufferSize = 64 * 1024 * 1024;
+  inline const static size_t AlignSize = 8;
```

Comment:
Maybe constexpr instead of/in addition inline to here?

Reply:
resolved